From 4752cd841ee4eb4911e371b0b6b524fc64941e86 Mon Sep 17 00:00:00 2001
From: Mianhan Liu <liumh1@shanghaitech.edu.cn>
Date: Thu, 30 Sep 2021 22:49:29 +0800
Subject: [PATCH] net/802/mrp.c: remove superfluous headers

mrp.c hasn't use any macro or function declared in linux/etherdevice.h and linux/netdevice.h.
Thus, these files can be removed from mrp.c safely without
affecting the compilation of the ./net module
---
 net/802/mrp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/net/802/mrp.c b/net/802/mrp.c
index 35e04cc53..5bc03320f 100644
--- a/net/802/mrp.c
+++ b/net/802/mrp.c
@@ -10,8 +10,6 @@
 #include <linux/kernel.h>
 #include <linux/timer.h>
 #include <linux/skbuff.h>
-#include <linux/netdevice.h>
-#include <linux/etherdevice.h>
 #include <linux/rtnetlink.h>
 #include <linux/slab.h>
 #include <linux/module.h>
-- 
2.25.1

